pkg/instance/ttlDestroyConsumer_test.go

```diff
@@ -17,17 +19,20 @@ import (
 func (s *ttlSuite) TestConsumeDeletesInstance() {
 	require := s.Require()
 
+	c := config.Config{UserService: config.Service{Username: "username", Password: "password"}}
+	uc := &userClient{}
+
 	consumer, err := rabbitmq.NewConsumer(
 		s.rabbitURI,
 		rabbitmq.WithConsumerPrefix("im-manager"),
 	)
 	require.NoError(err)
 	defer func() { require.NoError(consumer.Close()) }()
 
-	is := &instanceService{}
-	is.On("Delete", uint(1)).Return(nil)
+	is := &instanceService{userClient: uc}
+	is.On("Delete", "token", uint(1)).Return(nil)
```

Comment:
since you added a call to `userClient.SignIn(c.config.UserService.Username, c.config.UserService.Password)` in the implementation of Consume()

you will want to check that we actually signin using the correct credentials.

so I think something like

`uc.SignIn("user", "pw").Return("some token")`

Reply:
Do you mean...

	uc.On("SignIn", "username", "password").Return(&models.Tokens{
		AccessToken: "token",
	}, nil)

?